src/main/java/com/marklogic/mgmt/resource/security/CredentialsManager.java

```diff
@@ -15,29 +15,21 @@
  */
 package com.marklogic.mgmt.resource.security;
 
+import com.marklogic.mgmt.DeleteReceipt;
 import com.marklogic.mgmt.ManageClient;
-import com.marklogic.mgmt.SaveReceipt;
 import com.marklogic.mgmt.resource.AbstractResourceManager;
-import org.springframework.http.ResponseEntity;
 
 public class CredentialsManager extends AbstractResourceManager {
+
 	public CredentialsManager(ManageClient client) {
-		super(client);
+		super(client, true);
 	}
 
 	@Override
 	public String getResourcesPath() {
 		return "/manage/v2/credentials/properties";
 	}
 
-	@Override
-	protected String getCreateResourcePath(String payload) {
```

Comment:
I tested without this, and it doesn't appear needed - I think the Manager plumbing will figure out that it's JSON or XML and set the appropriate content type, which means "format" isn't needed.